labelbox/schema/project.py

```diff
@@ -63,6 +63,107 @@ def _validate_datetime(string_date: str) -> bool:
     return True
 
 
+def _build_filters(client, filters):
```

Comment:
NIT: Should this belong to project.py now that we're using it in project & dataset? Maybe this deserves a separate file

Reply:
^ Might be good to stick this into export_filters.py?